packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.1.2 - Jul 12 2021
+
+Incremental version for Teams Toolkit with following updates：
+
+Added:
+- UI Run button for debug and preview
+- Provide 'Report Issue' button in the pop-up error message to log issues when npm install fails
+
+On-going A/B testing:
```

Comment:
I think A/B testing is also a added feature? should we include it in the "Added" section?